### PR TITLE
virtual/httpd-cgi: add www-servers/caddy

### PR DESCRIPTION
**Pull request by AndCycle**
Closes: https://bugs.gentoo.org/868237

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: virtual/httpd-cgi/httpd-cgi-0-r5.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (ceamac) [inline virtual/httpd-cgi/httpd-cgi-0-r5.ebuild:4]:
```suggestion
EAPI=8
```

File: virtual/httpd-cgi/httpd-cgi-0-r5.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+DESCRIPTION="Virtual for CGI-enabled webservers"
+
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x64-solaris"
```
Comment (ceamac) [inline virtual/httpd-cgi/httpd-cgi-0-r5.ebuild:9]: Cannot push directly to stable, please run `ekeyword ~all httpd-cgi-0-r5.ebuild` (from `app-portage/gentoolkit`)

File: virtual/httpd-cgi/httpd-cgi-0-r5.ebuild [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2022 Gentoo Authors
```
Comment (ceamac) [inline virtual/httpd-cgi/httpd-cgi-0-r5.ebuild:1]:
```suggestion
# Copyright 1999-2024 Gentoo Authors
```

**Review by ceamac — COMMENTED** [review]
Thanks!

**Comment by AndCycle** [comment]
thanks for the review, I am new in this submit process.

**Review by ceamac — COMMENTED** [review]
Great.  Please squash the commits (and force push).

**Review by ceamac — APPROVED** [review]
I think you dropped the Closes: bug tag when squashing, I can add it back while merging.

Thank you!

**Pull request closed without merging**